acl_setup.php

```diff
@@ -542,7 +544,7 @@
         'admin'=>array('calendar', 'database', 'forms', 'practice', 'superbill', 'users', 'batchcom', 'language', 'super', 'drugs', 'acl','multipledb','menu','manage_modules'),
         'encounters'=>array('auth_a', 'auth', 'coding_a', 'coding', 'notes_a', 'notes', 'date_a', 'relaxed'),
         'lists'=>array('default','state','country','language','ethrace'),
-        'patients'=>array('appt', 'demo', 'med', 'trans', 'docs', 'notes', 'sign', 'reminder', 'alert', 'disclosure', 'rx', 'amendment', 'lab', 'docs_rm'),
+        'patients'=>array('appt', 'demo', 'med', 'trans', 'docs', 'notes', 'sign', 'reminder', 'alert', 'disclosure', 'rx', 'amendment', 'lab', 'docs_rm','pat_rep'),
         'sensitivities'=>array('normal', 'high'),
         'nationnotes'=>array('nn_configure'),
```

Comment:
This script is missing adding the new aco to the other ACLs(Accounting, Clinician, etc.)

Reply:
Fixed